android/samples/sample-gltf-viewer/src/main/java/com/google/android/filament/gltf/MainActivity.kt

```diff
@@ -83,7 +83,7 @@ class MainActivity : Activity() {
         modelViewer.loadModelGltf(buffer) { uri ->
             readCompressedAsset("models/$uri")
         }
-        modelViewer.transformToUnitCube()
+        modelViewer.transformToUnitCube(0.0f, 0.0f, 4.0f)
```

Comment:
👍

Reply:
Wait, this should probably take a `Float3` since ModelViewer already has a dependency on your math library.